en/customization/introduction.md

```diff
@@ -79,7 +79,27 @@ The translation service used has the [pricing](https://azure.microsoft.com/en-us
 The price for each 1 Million characters translated is 8.43 € and there is no fixed cost per month.
 Google and DeepL have an approximate price of between 16.00 € and 20.00 € for each 1 million characters and a fixed monthly cost.
 
-Alerts can be created on several parameters, including the `Number of translated characters` within the Azure administration panel in the **Supervision** section.
+Although technical measures have been taken to prevent misuse of this service, we recommend the creation of Alerts offered by Azure so that an Administrator can be notified in the event of detecting unusual use of the service.
+
+To create an Alert in Azure we must follow the following steps:
+1. Sign in to the **Azure Portal**.
+1. Register the resource providers needed to create Alerts
+  1. On the Azure portal menu, select **All services**.
+  1. In the **All services** box, enter **subscription**, and then select **Subscriptions**.
+  1. Select the subscription from the subscription list to view.
+  1. Select **Resource providers** and view the list of available resource providers.
+  1. Among the resource providers we must select **microsoft.insights** and click on the button **Register**
+1. Create new alert rule:
+  1. On the Azure portal menu, select **All services**.
+  1. In the **All services** box, enter **subscription**, and then select **Subscriptions**.
+  1. Select the subscription from the subscription list to view.
+  1. Select **Resources** and access the resource created from translations of the type **Cognitive Services**
+  1. In the **Monitoring** section select **Alerts** and we access **New alert rule**
+  1. Select the **Resource** about which we want to add the Alert (if we have followed the previous steps it should already be selected)
+  1. Add a **Condition**. In this case the one we are interested in has as its name `Characters Translated`. Once selected we must define the logic of the Alert to suit our needs. Ex: Operator (Greater than), Aggregation type (Total) y Threshold value(the number of characters we consider should be translated before being notified). In this section you can also set the time period and frequency of evaluation.
```

Comment:
> (Total) y Threshold

I guess we mean "(Total) and Threshold" :thinking:.

> has as its name

Sounds a bit funny to me :thinking:. How about "is called"?

Reply:
With this line:
`Ex: Operator (Greater than), Aggregation type (Total) y Threshold value(the number of characters we consider should be translated before being notified).`

I tried to explain that there is an "Operator" field in which the value "Greater than" must be selected, an "Aggregation type" field in which the value "Total" must be selected and a last "Threshold value" field in which a number of characters must be reported.
![Captura de pantalla 2020-02-27 a las 16 03 50](https://user-images.githubusercontent.com/16189/75456435-cf7a4380-597a-11ea-8d44-77957e53d39c.png)

New suggested text:
`Ex: Fill "Operator" field with "Greater than" option, fill "Aggregation type" field with 
"Total" option and fill "Threshold value" field with the number of characters we consider should be translated before being notified.`

Reply:
Sorry, I only meant we were writing "y" (in Spanish) instead of "and" (en English)! :grimacing:

Reply:
But the new text is better, so let's keep it :smile:.